### PR TITLE
[master] Traverse FCP devices and WWPN info to look for a proper path

### PR DESCRIPTION
**Pull request by changzhi1990**
In the current implementation, the script will use the first FCP and the first WWPN as the devNode.

But in some cases, files haven't been generated when we dedicate two FCP devices by executing smcli command. So this pr will traverse all FCP devices and all WWPNs to find a proper path.

### CODE REVIEW

File: zthin-parts/zthin/bin/refresh_bootmap [hunk 1]
```diff
@@ -490,15 +490,34 @@ function refreshFCPBootmap {
   # Set devNode according to whether multipath is enabled on the compute node
   wwid=""
   if [[ $multipath_enabled == 1 ]]; then
-      diskPath=/dev/disk/by-path/ccw-0.0.${fcps[0]}-zfcp-0x${wwpns[0]}:${lun}
-      wwid=`/usr/lib/udev/scsi_id --whitelisted $diskPath 2> /dev/null`
-      devNode="/dev/disk/by-id/dm-uuid-part1-mpath-$wwid"
-      inform "Multipath is enabled, using multipath devNode: $devNode."
+    # Traverse FCP devices and WWPN info to look for a proper path.
+    for i in "${!fcps[@]}"
+    do
```
Comment (dongyanyang) [inline zthin-parts/zthin/bin/refresh_bootmap:495]: no need to use seperate loop for multipath and singlepath?
just find one available diskPath before the if mutlpath_enabled.

**Comment by dongyanyang** [comment]
rebase & pep8/py27

**Comment by dongyanyang** [comment]
also PR description.

**Comment by changzhi1990** [comment]
pep8 pass:
[root@xyimgr77 python-zvm-sdk]# tox -e pep8
pep8 develop-inst-noop: /root/work/python-zvm-sdk
pep8 installed: DEPRECATION: Python 2.7 reached the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 is no longer maintained. A future version of pip will drop support for Python 2.7. More details about Python 2 support in pip, can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support,attrs==19.3.0,certifi==2020.4.5.2,chardet==3.0.4,configparser==4.0.2,contextlib2==0.6.0.post1,enum34==1.1.10,flake8==3.8.3,functools32==3.2.3.post2,idna==2.9,importlib-metadata==1.6.1,jsonschema==3.2.0,mccabe==0.6.1,netaddr==0.7.19,pathlib2==2.3.5,pycodestyle==2.6.0,pyflakes==2.2.0,PyJWT==1.7.1,pyrsistent==0.16.0,repoze.lru==0.7,requests==2.23.0,Routes==2.4.1,scandir==1.10.0,six==1.15.0,typing==3.7.4.1,urllib3==1.25.9,WebOb==1.8.6,zipp==1.2.0,-e git+https://github.com/openmainframeproject/python-zvm-sdk.git@456549f6a258aef0e36ccb2e8776bcfb0934c985#egg=zVMCloudConnector
pep8 run-test-pre: PYTHONHASHSEED='1040420445'
pep8 run-test: commands[0] | flake8
pep8 run-test: commands[1] | flake8 /root/work/python-zvm-sdk/scripts/sdkserver
_______________________________________________________ summary ________________________________________________________
  pep8: commands succeeded
  congratulations :)


py27 pass:
----------------------------------------------------------------------
Ran 640 tests in 0.960s

OK (skipped=3)
py27 run-test: commands[1] | python -m unittest discover -v -s /root/work/python-zvm-sdk/smtLayer/tests/unit
test_getVM_directory_py3 (test_vmUtils.SMTvmUtilsTestCase) ... ok
test_getDiskPoolSpace3390 (test_getHost.SMTGetHostTestCase) ... ok
test_getDiskPoolSpace3390Unknown (test_getHost.SMTGetHostTestCase) ... ok
test_getDiskPoolSpace3390UnknownWithFlag (test_getHost.SMTGetHostTestCase) ... ok
test_getDiskPoolSpace9336 (test_getHost.SMTGetHostTestCase) ... ok
test_getDiskPoolSpace9336Unknown (test_getHost.SMTGetHostTestCase) ... ok
test_getDiskPoolSpace9336UnknownWithFlag (test_getHost.SMTGetHostTestCase) ... ok
test_getReservedMemSize (test_makeVM.SMTMakeVMTestCase) ... ok
test_getReservedMemSize_equal_size (test_makeVM.SMTMakeVMTestCase) ... ok
test_getReservedMemSize_gap_G (test_makeVM.SMTMakeVMTestCase) ... ok
test_getReservedMemSize_invalid_suffix (test_makeVM.SMTMakeVMTestCase) ... ok
test_getReservedMemSize_max_less_than_initial (test_makeVM.SMTMakeVMTestCase) ... ok

----------------------------------------------------------------------
Ran 12 tests in 0.003s

OK
_______________________________________________________ summary ________________________________________________________
  py27: commands succeeded
  congratulations :)

**Comment by dongyanyang** [comment]
Need rebase @changzhi1990 
@bjhuangr @jichenjc pls help to review.

**Pull request merged**